include/qpid/dispatch/driver.h

```diff
@@ -142,12 +142,16 @@ void qdpn_driver_free(qdpn_driver_t *driver);
  * @param[in] driver driver that will 'own' this listener
  * @param[in] host local host address to listen on
  * @param[in] port local port to listen on
+ * @param[in] protocol family to use (IPv4 or IPv6)
```

Comment:
You should include 0 as a valid option as well.  It has a specific behavior.  Likewise on connector.